### PR TITLE
Update config-cli-subcommands-index.md

### PR DESCRIPTION
**Pull request by duhon**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Starting with version 2.2.6 was correctly work multi-threaded and added new indexer which support parallel mode

@shrielenee please merge it ASAP

### CODE REVIEW

**Comment by osrecio** [comment]
When Magento 2.2.6 will be released we will accept this PR. For the moment I will assign label " In Progress"

File: guides/v2.2/config-guide/cli/config-cli-subcommands-index.md [hunk 1]
```diff
@@ -120,23 +120,21 @@ Reindexing all indexers can take a long time for stores with large numbers of pr
 
 ### Reindex in parallel mode {#config-cli-subcommands-index-reindex-parallel}
 
-As of Magento 2.2.5, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
+As of Magento 2.2.6, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
 
 The following indexes can be run in parallel mode:
 - Catalog Search Fulltext can be paralleled by store views.
 - Category Product can be paralleled by store views.
+- Catalog Price can be paralleled by website and customer groups.
```
[the remark this answers is not in the document]
Comment (duhon) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-index.md]: done

**Comment by duhon** [comment]
@osrecio I updated content, please check again this PR before you merge

**Comment by vzabaznov** [comment]
@osrecio Hi 2.2.6 was released, please merge this PR asap

**Comment by shrielenee** [comment]
Thank you @duhon and @vzabaznov and @osrecio!

**Pull request merged**